airflow/contrib/hooks/gcp_cloud_build_hook.py

```diff
@@ -48,7 +48,7 @@ class CloudBuildHook(GoogleCloudBaseHook):
     :type delegate_to: str
     """
 
-    _conn = None
+    _conn = None  # type: Optional[Resource]
```

Comment:
This is not a proper type. Google API Client (discovery based) generate types at runtime. I think, we should use `Any` or introduce generated prototype. I prefer `Any`. The Resource class doesn't have public method. WDYT?